src/content.js

```diff
@@ -551,6 +551,25 @@ function init() {
 			event.preventDefault();
 			indentInput(event.target);
 			return false;
+		} else if (event.which === 13 && event.shiftKey && (event.ctrlKey || event.metaKey)) {
```

Comment:
I’ve seen new event properties that would allow us to just use `=== ‘Enter’` instead of numbers, can you try this?

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
Switched to using `event.key`.